2_dataCleaning/CleaningChallenge_Code.Rmd

```diff
@@ -44,41 +49,69 @@ hist(subjects$Q2)
 hist(subjects$Q3)
 table(subjects$Condition)
 
-#Condition variable has a missing data point for sub 9. README says they should be in condition C
+#check for missing values
+subjects[!complete.cases(subjects),]
+
+```
+
+## Correcting data issues
+
+Remember to check the README for additional information on the data that may help you decide what needs to be fixed.
+
+```{r}
+
+#Condition variable has a missing data point for sub 9. 
+
 subjects$Condition[subjects$ID==9] <- "C"
```

Comment:
should we do this using traditional bracket indexing or mutate? either works since this is a one-off thing